SDAPackage/SDA/moving_obstacle.py

```diff
@@ -0,0 +1,26 @@
+from Obstacle import *
+from SDA import Constants
+
+class MovingObstacle(Obstacle)
+	
+
+	def __init__(self, point, radius, height, velocity)
+		super(MovingObstacle, self).__init__(point, Constants.STATIONARY_OBSTACLE_SAFETY_RADIUS)
```

Comment:
This shouldn't be STATIONARY_OBSTACLE_SAFETY_RADIUS because it is not for stationary obstacles.